### PR TITLE
Enable integration tests on Travis

### PR DESCRIPTION
**Pull request by alxndrsn**
These weren't enabled on Travis before.  I'm not entirely happy with the two additions to `.travis.yml`, but I can't think of a neater way to do them except creating another `exec` target in `Gruntfile.js` and renaming the other integration-test-related targets better.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +17,7 @@ before_script:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - curl -X PUT localhost:5984/medic
+  - curl -X PUT http://localhost:5984/_config/admins/admin -d '"pass"'
```
Comment (garethbowen) [inline .travis.yml]: We have a grunt task to do this: `exec:setupAdmin`

**Comment by garethbowen** [comment]
We already `kanso push` a db and set up an admin for e2e. It'd be good to either reuse the db or at least reuse the db creation commands.

Currently some setup is done in the Gruntfile and some in the setup for the e2e tests. This PR adds some setup to the .travis file. It's all getting a bit complicated.

I think we should rework it so deleting the db (if it exists), creating the db (kanso push), and creating the admin user are all in Grunt so e2e and migration testing can use the same commands.

**Comment by garethbowen** [comment]
Looks good. Feels like a really clean solution now. The API branch is failing on jshint stuff - you probably need to define a child `.jshintrc` file, eg: https://github.com/medic/medic-api/blob/master/public/login/.jshintrc

Fix that and merge both when ready!

**Comment by alxndrsn** [comment]
Merged manually.

**Comment by mandric** [comment]
@alxndrsn if the branch is merged/not needed, please delete.

**Comment by mandric** [comment]
Thanks

**Pull request closed without merging**